task4_1.py

```diff
@@ -0,0 +1,15 @@
+# a recursive function which returns the nth Fibonacci number
+ans = [0 for i in range(1000)]
```

Comment:
Please use dictionary for caching instead of list.

Reply:
It is done.